jbdevstudio/com.jboss.jbds.central.discovery/plugin.xml

```diff
@@ -43,6 +43,33 @@ reasonable, reporting issues to these providers as required.</description>
 
       <!-- use connectorCategory relevance value to sort from largest (80) to smallest (20) -->
 
+      <connectorCategory
+            description="Patches"
+            id="org.jboss.tools.central.discovery.a.a.patches"
+            name="Patches"
+            relevance="90">
+         <group id="org.jboss.tools.central.discovery.a.a.patches" /> 
+      </connectorCategory>
+
+      <connectorDescriptor
+            categoryId="org.jboss.tools.central.discovery.a.a.patches"
+            groupId="org.jboss.tools.central.discovery.a.a.patches"
+            description="Web Tools 3.6.3 Patch Feature"
```

Comment:
We already have "Web Tools 3.6.3 Patch Feature" as a name. Maybe we could use the description to show some details. For example: "Fixes Eclipse freeze for some JavaScript projects", or something like that. WDYT?